src/components/CardList/Card/Card.tsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import styles from "./Card.module.scss";
+import EmptyCase from "@/assets/svg/CardImageEmptyCase";
+import getElapsedTime from "../../../hooks/getElapsedTime";
+import getFormatDate from "../../../hooks/getFormatDate";
+import { LinkProps } from "@/constants/index.types";
+
+interface CardProps {
+  link: LinkProps;
```

Comment:
타입은 이렇게 표현되는게 좋을 것 같아요~
-Props라는 표현은 컴포넌트에 한정된 의미를 갖고 있고, 링크라는 것은 데이터 그 자체니까요!
```suggestion
  link: Link;
```